package.json

```diff
@@ -20,8 +20,13 @@
     "bundle": "npm run build:directive && npm run build:component"
   },
   "dependencies": {
+    "begeta": "^1.0.0",
+    "buefy": "^0.7.0",
+    "highlight.js": "^9.13.1",
     "to-style": "^1.3.3",
-    "vue": "^2.5.11"
+    "vue": "^2.5.11",
+    "vue-embed": "^1.0.0",
+    "vue-highlight-component": "^1.0.0"
```

Comment:
เอาออกจาก dependencies ไปด้วย ให้อยู่แต่ใน devDependencies ฝากเอา to-style ไปด้วยครับ

Reply:
อ๋อๆ เดี๋ยวเย็นๆ เอาออกให้ครับทำงานก่อนครับ 555